package.json

```diff
@@ -67,7 +67,10 @@
     "@babel/plugin-proposal-export-default-from": "7.10.1",
     "@babel/plugin-proposal-export-namespace-from": "7.10.1",
     "@babel/types": "7.6.x",
+    "@react-native-community/blur": "^3.6.0",
```

Comment:
I don't see this being used?

Reply:
It's not - but it's a peer dependency of react-native-ui-lib which we had to upgrade. I hope we can get rid of this soon in their v6